.env.example

```
@@ -4,4 +4,4 @@ AUTO_DELETE_CHAT=true
 PUBLIC_BASE_URL=your_external_base_url_here
 SECURE_1PSID=your_psid_value_here
 SECURE_1PSIDTS=your_psidts_value_here
-API_KEY=your_api_key_here
+API_KEY=Gemi2Api-Server
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The API_KEY key should go before the AUTO_DELETE_CHAT key

(UnorderedKey)